ci.sh

```diff
@@ -161,16 +152,21 @@ build_qemu() {
     --disable-parallels \
     --disable-qcow1 \
     --disable-qed \
+    --disable-replication \
     --disable-sdl \
     --disable-smartcard \
+    --disable-snappy \
     --disable-usb-redir \
     --disable-user \
+    --disable-vde \
     --disable-vdi \
     --disable-vnc \
     --disable-vvfat \
     --disable-xen \
     --disable-lzo \
+    --disable-zstd \
     --enable-lto \
+    --enable-slirp=git \
```

Comment:
What does this do?

Reply:
force qemu to use its internal copy of libslirp, even if there is an external one, there is no need to add it to the dependency list and as a static library to link with which alpine doesn't provide yet

Reply:
slirp is what allows `netdev user` to work, so has been there all along from earlier releases.

what has changed recently is that it has been made an independent project with releases of its own, and so it is expected to be used eventually as an external dependency (ex: `apk add libslirp`).

without this, if the builder has libslirp installed, meson will detect it and link with it, which is AFAIK not the intended outcome; and while the static library could be added to the LDFLAGS for macOS, there is no way to do the same in alpine yet.

Reply:
I see, thanks.